### ISSUE TITLE
Question about the Choice of Training Epochs

### ISSUE THREAD

**Issue by ShacklesLay** [ContextualAI/CLAIR_and_APO#1]
I've been exploring your fascinating work on "Anchored Preference Optimization and Contrastive Revisions: Addressing Underspecification in Alignment" and appreciate the effort you've put into this work.

I noticed that the number of epochs is set to 18, which seems unusually high.  Could you please provide some insights into the reasons behind setting the epoch count to 18 for this particular work? Understanding this could greatly help in my learning and application of your approach.

Thank you for your time and assistance!

**Comment by karel-contextual** [comment]
Hi @ShacklesLay , thanks for expressing interest in our work!

Setting training hyperparameters is always a challenging problem. We decided to go for a lower learning rate and high number of epoch to get a more stable run. We did not sweep the number of epochs, 18 was just an incidental choice. A longer training also helped us see the difference between DPO and APO better, as we outline in figure 4 of the paper.

I'll aim to release the full evaluation results and wandb logs soon, this may also help you pick these hyper parameters.
